frontend/service.go

```diff
@@ -62,19 +59,19 @@ func (s *LwdStreamer) GetAddressTxids(addressBlockFilter *walletrpc.TransparentA
 
 	params[0] = json.RawMessage(st)
 
-	result, rpcErr := s.client.RawRequest("getaddresstxids", params)
+	result, rpcErr := common.RawRequest("getaddresstxids", params)
 
 	// For some reason, the error responses are not JSON
 	if rpcErr != nil {
-		s.log.Errorf("Got error: %s", rpcErr.Error())
-		return nil
+		common.Log.Errorf("Got error: %s", rpcErr.Error())
+		return err
```

Comment:
This (and a few similar cases below) was just broken, `nil` means no error by convention.

Reply:
Were these maybe intentional for some reason, like to hide the specific error message from the client or something? (I don't think so, it looks like it was just broken)